### ISSUE TITLE
Electron 11.0.2 vibrancy error

### ISSUE THREAD

**Issue by 15814596022** [electron/electron#26642]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 11.0.2
* **Operating System:**
  * macOS 10.13.6
* **Last Known Working Electron version:**
  * 10.1.5

### Expected Behavior

Frosted glass effect

### Actual Behavior

```
Electron[3154:38216] *** Assertion failure in -[NSVisualEffectView _internalMaterialFromMaterial:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1561.60.100/AppKit.subproj/NSVisualEffectView.m:765
```

### To Reproduce
```javascript
let win = new BrowserWindow({
  titleBarStyle: 'hidden',
  visualEffectState: 'followWindow',
  frame: false,
  show: false
})
win.loadURL('https://www.electronjs.org/')
win.on('ready-to-show', () => {
  win.show()
})
```

**Comment by seunghwanbae** [comment]
I have the same problem.
Electron 11.0.3
Mac OS 10.13.6

**Command Error**

```
Electron[2807:41085] *** Assertion failure in -[NSVisualEffectView _internalMaterialFromMaterial:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/AppKit/AppKit-1561.61.100/AppKit.subproj/NSVisualEffectView.m:765
/Users/home/.nvm/versions/node/v14.15.1/lib/node_modules/electron/dist/Electron.app/Contents/MacOS/Electron exited with signal SIGILL
GVA error: Not detecting IGPU in IORegistry!
GVA error: Not detecting valid offline codec!
```

**Comment by codebytere** [comment]
@15814596022 does this happen with other visual effect states? I'm concerned this is just an SDK bug on 10.13

**Comment by 15814596022** [comment]
> @15814596022 does this happen with other visual effect states? I'm concerned this is just an SDK bug on 10.13

visualEffectState
 - followWindow
 - active
 - inactive 

I tried them all and got same result
Maybe just on 10.13, not many people ask this question yet

**Comment by pdesantis** [comment]
This happens to me with `vibrancy: 'hud'` on 10.13

**Comment by codebytere** [comment]
@pdesantis are you sure? 'hud' is only available on 10.14 and above - we explicitly gate it as such:

https://github.com/electron/electron/blob/master/shell/browser/native_window_mac.mm#L1573-L1593

and yeah - unfortunately folks i'm fairly certain this is an SDK bug on 10.13 as we didn't change anything in that region of the code between 10 and 11 except the SDK. As such I'm closing this out as a `wontfix` but you are welcome to file a radar with Apple.

**Comment by 15814596022** [comment]
> @pdesantis are you sure? 'hud' is only available on 10.14 and above - we explicitly gate it as such:
> 
> https://github.com/electron/electron/blob/master/shell/browser/native_window_mac.mm#L1573-L1593
> 
> and yeah - unfortunately folks i'm fairly certain this is an SDK bug on 10.13 as we didn't change anything in that region of the code between 10 and 11 except the SDK. As such I'm closing this out as a `wontfix` but you are welcome to file a radar with Apple.

thx!
I try vibrancy state which outside @available(macOS 10.14, *) is ok
